src/core/processors/canvasprocessor.cpp

```diff
@@ -46,29 +46,56 @@
 
 namespace inviwo {
 
-CanvasProcessor::CanvasProcessor()
+CanvasProcessor::CanvasProcessor(InviwoApplication* app)
     : Processor()
     , inport_("inport")
     , inputSize_("inputSize", "Input Dimension Parameters")
     , dimensions_("dimensions", "Canvas Size", size2_t(256, 256), size2_t(1, 1),
-                  size2_t(4096, 4096), size2_t(1, 1), InvalidationLevel::Valid)
+                  size2_t(10000, 10000), size2_t(1, 1), InvalidationLevel::Valid)
```

Comment:
Was there a particular reason to increase this? Not possible to display it on any screen

Reply:
two 4k monitors is wider than 4096, this is just the max value

Reply:
my setup is 8k wide...